### PR TITLE
Fixed Issue 1104

### LINKED ISSUE #1104: configure-integration-tests tries to remove a value right after I set it
This flow around optional values seems counter-intuitive:

>
**[value] found as the configured application Secret
Want to change this? Press Y, otherwise we'll move on:** y
**Change the application Secret to use for the integration tests:** [new-value]
**Want to remove this optional value, press Y:**
```

If the user has pressed `Y` here, `Want to remove...` shouldn't be an option...

### PR DESCRIPTION
**Pull request by Anubhav10**
Made changes in .\script\configure-integration-tests.ps1.
For an optional value it gives the option to change or remove the value, for a necessary value it doesn't give an option.

Fixes #1104

### CODE REVIEW

**Comment by shiftkey** [comment]
While we're in here, could you look at making the whitespace consistent? It's probably my fault that there's some inconsistencies in here...

File: script/configure-integration-tests.ps1 [hunk 1]
```diff
@@ -43,21 +43,34 @@ function VerifyEnvironmentVariable([string]$friendlyName, [string]$key, [bool]$o
...
+	        	elseif ($reset -eq "R")
+		        { 
+        			SetVariable $key $null
+	        	}
+             }
+            
+        }
+	   else
+	   {
+	    $reset = Read-Host -Prompt "Want to change this value , press Y, otherwise we'll move on"
```
Comment (shiftkey) [inline script/configure-integration-tests.ps1]: Let's tweak this message: `Press Y to change this value, otherwise we'll move on"

**Comment by shiftkey** [comment]
@Anubhav10 this looks good - just that one improvement to the message and some cleanup of the whitespace and I think this is good to go!

File: script/configure-integration-tests.ps1 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿# TODO: this should indicate whether a variable is required or optional
+# TODO: this should indicate whether a variable is required or optional
```
Comment (shiftkey) [inline script/configure-integration-tests.ps1]: This is an old comment from before I actually implemented the optional parameters. Feel free to remove this :fire:

**Comment by Anubhav10** [comment]
I'll do the suggested changes. Yesterday, the Travis CI build was failing, how come it passed today? I tried to understand why it was failing but I couldn't.

**Comment by ryangribble** [comment]
Unfortunately we are sometimes seeing unrelated failures on the linux and/or OSX travis CI builds.
#1076

**Comment by Anubhav10** [comment]
@shiftkey The whitespace inconsistency was in my code and was my fault. I have removed the TODO comment and changed the message.

**Comment by shiftkey** [comment]
@Anubhav10 excellent, thanks for this!

**Pull request merged**